public/i18n/en/translation.json

```diff
@@ -1792,7 +1796,37 @@
       "account_settings_for": "Account settings for '{name}'",
       "change_password": "Change password",
       "language": "Language",
-      "ui_language": "User interface language"
+      "ui_language": "User interface language",
+      "change_language": "Change language",
+      "change_password_description": "Modify your authentication password for accessing the controller's web interface.",
+      "ssh_key": "SSH key",
+      "ssh_key_description": "An SSH key pair consists of a public key (shared with servers) and a private key (kept secure locally), enabling secure authentication and data encryption in SSH connections without the need for passwords.",
+      "two_factor_authentication": "Two-factor authentication",
```

Comment:
Can we remove 2FA i18n strings? They are not currently used, and they'll be probably placed inside a distinct i18n section